packages/flutter/lib/src/material/material.dart

```diff
@@ -89,6 +89,8 @@ abstract class InkHighlight {
 
   /// The color use to visually represent the emphasis.
   Color get color;
+
+  /// The color use to visually represent the emphasis.
```

Comment:
This grammar is dubious, but seems to be copy pasta of other bad grammar.

Also, no need to doc the setter.